normaliz.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@prefix@/include
+
+Name: @PACKAGE_NAME@
+Description: tool for discrete convex geometry: affine monoids, vector configurations, rational polyhedra and cones
+Version: @PACKAGE_VERSION@
+Libs: -L${libdir} -libnormaliz
```

Comment:
Should there be an openmp flag here?

Reply:
Hi Mahrud, 

In `CXXFLAGS`? In `LDFLAGS`?

pkg-config does not have a dedicated C++ compiler options (for CXXFLAGS) parameter, only `-cflags`.
I don't know whether it's normal to put C++ options there, or they should be  put in a separate `foobar` variable in .pc file, which can be inspected with `pkg-config --variable foobar` ?

Reply:
OpenMP is also usable with C. For reference, this is what fflas-ffpack has in their pkgconfig file:
```
Cflags: -I${prefix}/include    -fopenmp
```
I don't care either way, I was just wondering.

Reply:
OK, I have to figure out how to extract this flag, whenever it is set, from autotools.

Reply:
Not to make this more complicated, but I think in some cases this line needs to also include some of `-leantic -leanticxx -lflint -lnauty -lgmp`, or better yet the pc file could use the `Requires` field (e.g. see [here](https://people.freedesktop.org/~dbn/pkg-config-guide.html)).